readthedocs/builds/tasks.py

```diff
@@ -78,19 +77,12 @@ def route_for_task(self, task, args, kwargs, **__):
         return
 
     def _get_version(self, task, args, kwargs):
-        if task == 'readthedocs.projects.tasks.update_docs_task':
-            build_pk = kwargs.get('build_pk')
-            try:
-                build = Build.objects.get(pk=build_pk)
-                version = build.version
-            except Build.DoesNotExist:
-                log.info(
-                    'Build does not exist. Routing task to default queue. build_pk=%s',
```

Comment:
Why did you remove this check?

Reply:
Both tasks pass the `version_pk` as the first parameter, so there no need to get the build.